client/src/interface/Board.ts

```diff
@@ -14,3 +14,8 @@ export interface BoardApiData {
   boards: Array<IBoard>;
   error?: { message: string };
 }
+
+export interface NewBoardApiData {
+  board: IBoard;
+  error?: { messge: string };
+}
```

Comment:
I actually deleted this file in #63 as it was posing inconsistent with `src/context/types/kanban.ts` and I wanted to ensure we didn't have issues with types/interfaces

Reply:
Sorry, can you explain it a bit further?

Reply:
Of course! In this [commit](https://github.com/hatchways/team-rambo/pull/63/commits/d9717e6f5ca94326a4859107e9e541ad245c0ad0), I deleted IBoard/Card/Column and refactored to match the interfaces living in `src/context/types/kanban.ts`. See attached screenshot (left is old and deleted, right is current interfaces) with differences!

![Screen Shot 2021-05-27 at 9 37 41 PM](https://user-images.githubusercontent.com/35545740/119930162-c5efda80-bf33-11eb-910d-1dba8ee656d8.png)

Reply:
That makes a lot of sense. Thanks!

Reply:
I've phased out the types folder as it was a mistake on my part for not noticing the interfaces folder. It should all be in the main branch now. 👍🏻